### PR TITLE
Update the description of the ticket to point at RFC 1721

### LINKED ISSUE #70538: Odd "error[E0658]: is unstable" diagnostic
```rust
#[link(name = "c", cfg(yes))]
extern {}
```

```
error[E0658]: is unstable
 --> src/lib.rs:1:1
  |
1 | #[link(name = "c", cfg(yes))]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: for more information, see https://github.com/rust-lang/rust/issues/37406
```

:thinking: 

In addition to the malformed diagnostic, the tracking issue has been closed already. According to https://rust-lang.github.io/rfcs/1721-crt-static.html#lazy-link-attributes, this is supposed to be a permanently-unstable implementation detail. In that case, should the link be updated to point somewhere else?

This issue has been assigned to @Rustin-Liu via [this comment](https://github.com/rust-lang/rust/issues/70538#issuecomment-606399105).

### PR DESCRIPTION
**Pull request by 0xPoe**
Fixes #70538.

My first PR to rust. So please let me know if I'm doing something wrong.

### CODE REVIEW

**Comment by jonas-schievink** [comment]
@Rustin-Liu I think the text should be changed to actually point out what's unstable. The link you added can be added to the tracking issue #37406 instead (someone with editing permissions can add it).

**Comment by Rustin-Liu** [comment]
@estebank could you please take a look?

File: src/librustc_metadata/native_libs.rs [hunk 1]
```diff
@@ -162,8 +162,13 @@ impl Collector<'tcx> {
             }
         }
         if lib.cfg.is_some() && !self.tcx.features().link_cfg {
-            feature_err(&self.tcx.sess.parse_sess, sym::link_cfg, span.unwrap(), "is unstable")
-                .emit();
+            feature_err(
+                &self.tcx.sess.parse_sess,
+                sym::link_cfg,
+                span.unwrap(),
+                "kind=\"link_cfg\" is unstable, see https://github.com/rust-lang/rfcs/pull/1721",
```
Comment (varkor) [inline src/librustc_metadata/native_libs.rs:169]: I don't think we need to mention https://github.com/rust-lang/rfcs/pull/1721 here — we already mention the relevant issue below. However, https://github.com/rust-lang/rust/issues/37406 seems to be closed, so maybe we'll need to open it.
```suggestion
                "kind=\"link_cfg\" is unstable",
```

**Review by varkor — CHANGES_REQUESTED** [review]
@Rustin-Liu: after the suggested change, could you flatten your commits into one? Then everything looks good! I think we should be pointing to the tracking issue, but RFC 1721 should be more prominently displayed: I'll try to sort that out.

File: src/librustc_metadata/native_libs.rs (hunk 1 above)
Comment (Rustin-Liu) [reply src/librustc_metadata/native_libs.rs]: Removed.

**Pull request merged**